website/content/maintainers/github/templates/required/governance-subprojects.md

```diff
@@ -0,0 +1,290 @@
+the "Organization Member" level on our template [Contributor Ladder], or
+whatever your project equivalent is. 
+
+Depending on your project's needs, you may want to have alternative or additional
+types of representatives. For example, you might want to have a representative
+elected at large by all collective maintainers, which can be a way to ensure
+that your biggest subprojects get additional representation. Text used for that
+would be:
+
+```
+The Maintainer Representative(s) will be elected by the collective Maintainers
+of all subprojects, as defined in the Contributor Ladder.  They will be elected 
+annually.
+```
+
+In the current text, candidates are not required to be Organization Members 
+or Maintainers themselves. You may or may not want to require that.  If you 
+do, use text like:
+
+```
+The Contributor Representative(s) will be elected by the collective Organization
+Members of all subprojects, as defined in the Contributor Ladder.  
+Representative(s) must be Organization Members themselves, and will be elected 
+annually.
+```
+
+### Code of Conduct Committee
+
+One of the key duties of the SC is to appoint the Code of Conduct Committee (CoCC),
+who are responsible for handling any CoC violation reports.  This assumes that
+you have a large enough pool of maintainers and other very involved contributors
+to staff a CoCC outside of the SC.  If you do not, use instead the following text:
+
+```
+## Code of Conduct Reports
+
+The Steering Committee is responsible for handling initial Code of Conduct
+reports or incidents. They will receive reports of conduct violations 
+confidentially.  If a report is determined to be a violation, they will 
+recommend action on it appropriate to the scale, nature, and context of the 
+violation, from requiring an apology, up to expulsion of an individual from the 
+project. Should any report require a full investigation, or involve a member
+of the Steering Committee themselves, the report will be forwarded to the
+CNCF conduct team for handling.
+```
+
+If your CoCC is defined in a separate document, delete the text and link to it
+instead.
+
+### Adding and Removing Subprojects
+
+These sections outline the procedure for adding a new subproject, as well as 
+removing one. There is not a lot of need for customization here, unless
+you want to specifically spell out additional technical requirements.  The
+one thing you need to spell out is the location of your "archive" or "attic"
+namespace.
+
+The one paragraph you may want to consider is the one outlining Experimental
+Subprojects.  This provision is there because sometimes you want to "try" a
+subproject, but it currently doesn't have useful code or a pool of maintainers.
+In that case, you may want to give that subproject a route to join your project
+without the same expectations and authority as the established subprojects.
+If this doesn't sound like something you're ever likely to do, you can cut this
+paragraph.
+
```

Comment:
It would be great if we can link to examples of this template if it's used by any projects already.